configs/components/oifs/oifs.yaml

```diff
@@ -139,6 +140,15 @@ compile_infos:
             install_bins: make/esm/oifs/bin/oifs
             destination: oifs-43r3
             with_xios: true
+          43r3-awicm-3.1.1:
+            requires:
+            - oasis3mct-4.0-awicm-3.1
+            branch: awicm-3.1.1
+            comp_command: "export OIFS_TOPLEVEL_DIR=${model_dir}; export OIFS_XIOS=enable ; export OIFS_XIOS_DIR=${model_dir}/../xios ; export OIFS_XIOS_INCLUDE=-I/${model_dir}/../xios/inc/; cd make; ../fcm/bin/fcm make -v -j8 -f oifs.fcm ; chmod -R ${source_code_permissions} .; mv esm/oifs/bin/master.exe esm/oifs/bin/oifs"
+            git-repository: https://gitlab.dkrz.de/ec-earth/oifs-43r3.git
+            install_bins: make/esm/oifs/bin/oifs
+            destination: oifs-43r3
+            with_xios: true
```

Comment:
is this different from other versions (in particular the `comp_command`)? If no, we should just make it universal IMHO

Reply:
the comp command has changed in the past, e.g. with XIOS integration. It will change further e.g. with single precision.